src/SFA.DAS.EmployerIncentives.Domain.UnitTests/ValueObjects/WhenHasSignedRequiredAgreementVersion.cs

```diff
@@ -10,28 +10,63 @@ namespace SFA.DAS.EmployerIncentives.Domain.UnitTests.ValueObjects
     [TestFixture]
     public class WhenHasSignedRequiredAgreementVersion
     {
-        private List<IncentivePaymentProfile> _incentivePaymentProfiles;
+        private IncentivesConfiguration _config;
```

Comment:
Why change this?  Should still be  List<IncentivePaymentProfile>.

Reply:
encapsulation